Field_Simulation/html/example.html

```diff
@@ -0,0 +1,98 @@
+<html>
+<head></head>
+<body>
+	<canvas id='bitmap' width ='400' height = '400'></canvas>
+	<script>
+		var time;
+		var CIRCLE_TIME = 1 * 60 * 200;
+		var CIRCLE_RADIUS = 50;
+		var FIELD_SIZE = 75;
+
+		start();
+		setInterval(loop,100);
+
+		function start()
+		{
+			time = new Date().getTime();
+		}
+		function loop()
+		{
+			var element = document.getElementById("bitmap");
+			var bitmap = element.getContext("2d");
+
+			// Clear previous vectors
+			bitmap.clearRect(0,0,element.width,element.height);
+			console.log(element.width,element.height);
+
+			// Gets the circular field vector
+			var angle = 360 * get_ratio();
+			angle = radians(angle);
+			var circle = [Math.sin(angle)*CIRCLE_RADIUS,Math.cos(angle)*CIRCLE_RADIUS];
+			var zero = [element.width / 2, element.height / 2];
+			var vector = circular_field(angle);
+
+			// Makes adjustments between vectors and canvas points
+			circle[0] += zero[0]
+			circle[1] = (circle[1]+zero[1]);
+			vector[0] += circle[0];
+			vector[1] = (vector[1]+circle[1]);
+			// Draws the vector
+			bitmap.beginPath();
+			bitmap.strokeStyle = "blue";
+			bitmap.moveTo(circle[0], circle[1]);
+			bitmap.lineTo(vector[0], vector[1]);
+			bitmap.stroke();
+			bitmap.strokeStyle = "black";
+			// Draws the circle
+			bitmap.beginPath();
+			bitmap.arc(zero[0], zero[1], CIRCLE_RADIUS, 0, 2 * Math.PI);
+			bitmap.stroke();
+		
+		}
+		function clear_lines()
+		{
+			while(mutex);
```

Comment:
Definitely a bad approach. Fix that with normal timeouts or interval callbacks.

Reply:
That function is entirely irrelevant and I will remove it